src/deployment/linux.md

```diff
@@ -342,10 +314,18 @@ To use the LXD container backend:
 $ snapcraft --use-lxd
 ```
 
-## Publish
+Once the `snapcraft.yaml` file is complete,
```

Comment:
It feels odd to have a title as a continuation of a sentence. Can we turn this into a complete sentence?

Reply:
Or just delete this line altogether, perhaps? It's really just leading into the next two sections, and their titles provide pretty much all the reader needs to know.

Reply:
I agree it would be better to remove the line.